docs/user/mcdoc/index.adoc

```diff
@@ -1001,12 +1001,12 @@ which will get replaced by actual types when the type alias is instantiated else
 type NumericRange<T> = ( <1>
 	T | <2>
 	[T, T] | <2>
-	struct { min: T, max: T }, <2>
+	struct { min: T, max: T } <2>
```

Comment:
Trailing commas are allowed and even recommend in vanilla-mcdoc, so I'd leave this

Reply:
Isn't this is a union type though? A trailing pipe would be valid but I'm not sure about a trailing comma.

Reply:
Oh good point yes